remoteappmanager/rest/exceptions.py

```diff
@@ -0,0 +1,19 @@
+class BaseException(Exception):
```

Comment:
Please rename There is already a python Exception named BaseException. I would suggest RESTException